src/components/events/EventCard.tsx

```diff
@@ -9,16 +9,16 @@ interface EventCardProps {
 
 const EventCard: React.FC<EventCardProps> = ({ image, text, title }) => {
   return (
-    <div className="flex w-[21vw] flex-col items-center rounded-[3vw] bg-white px-[3%] pb-[7%] pt-[0.5%] shadow-2xl">
+    <div className="mx-auto flex w-[50vw] flex-col items-center rounded-[3vw] bg-white px-[3%] pb-[7%] pt-[0.5%] shadow-2xl md:w-[80%] md:rounded-[2.5vw] md:py-[5%]">
       <Image
         src={image}
         alt="InfoPicture"
         className="aspect-[4/3] object-contain"
       />
-      <p className="font-title text-[2.7vw] font-extralight text-hearts-blue">
+      <p className="md:text-xxl font-title text-[2.7vw] font-extralight text-hearts-blue">
         {title}
       </p>
-      <p className="w-[90%] text-[0.8vw] font-extralight text-hearts-brown">
+      <p className="w-[90%] text-[0.8vw] font-extralight text-hearts-brown md:text-[1vw]">
```

Comment:
md: should go before the old formatting you want to keep for desktop, new mobile formatting has no prefix